shrinkwrap.py

```diff
@@ -323,7 +323,7 @@ def charm_channel(app, charm_path) -> str:
             channel = "auto"
 
     if channel == "auto":
-        channel = "latest/stable"
+        channel = "stable"
```

Comment:
Honestly... this is only "better" -- but not perfect

i had a discussion with some folks on the snap-store team that gave me this advice.  it seems 
`snap-store-proxy fetch-snaps <snap> --channel=latest/stable`
*should* work, but theres a [bug](https://bugs.launchpad.net/snapstore-snap/+bug/1939063)

`snap-store-proxy fetch-snaps <snap> --channel=stable` is the work around. 

here's what we should do when getting the snap, but this PR doesn't include since it's a work-around
1) look in the layer.yaml for the snap and which channel is the default
2) look in the charm config for a `channel` option
    1) if that config is 'auto'  -- return to the default from (1)
3) look in the bundle config for an over ride

It seems the 'etcd' charm has this option `auto` which I've not seen in any other charm where it uses the channel "3.4/stable" hard coded in the charm code. 

Anyway -- this change is to get around the above LP#1939063